### ISSUE TITLE
MaterialCheckBox ReadOnly (added)

### ISSUE THREAD

**Issue by valimaties** [leocb/MaterialSkin#217]
Hi.
I have downloaded project in my Visual Studio and I have made a little "refreshing" on `MaterialCheckBox` object, because I needed `ReadOnly` option. 

So, In `MaterialCheckBox.cs` file I have added in properties:
```csharp
[Browsable(true)]
public bool ReadOnly { get; set; }

private CheckState _oldCheckState;
```

In `OnCreateControl` event I have added these lines of code:
```csharp
// In MouseEnter event added line:
 _oldCheckState = CheckState;

// In MouseDown, MouseUp, KeyDown, KeyUp events I have added line:
if (ReadOnly) CheckState = _oldCheckState;
```

This is how it works:
![CheckBoxReadOnly](https://user-images.githubusercontent.com/49796453/134306975-b8d651c5-4f7a-47b4-a6a3-5769f9305bf0.gif)

Regards,
Vali

**Comment by orapps44** [comment]
Hi,

Thank you for you're work. It looks good.

Did you consider to user Enabled property instead ?
From my point of view Enabled property helps user to see a control isn't active for modification.

However if otheres are interested by read-only property, I suggest you to prepare a Pull request.

**Comment by valimaties** [comment]
Thank you, @orapps44 .
I've considered Enabled property, but disabled color is not so friendly and not so visible as the colored object. I have made some applications and user's feedback was to not use that object as disabled, but to make them "unchangeable". So I added ReadOnly property in CheckBox object and they are happy now.

I'm really not so familiarized with github options, I don't know how to make pull requests. That's why I've posted code here...

Regards,
Vali
